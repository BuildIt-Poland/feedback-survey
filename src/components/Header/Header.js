import React from 'react';
import styled from 'styled-components';

import { Link as LinkBase } from 'react-router-dom';

import { colorDarkBlue } from '../../styles/designTokens';
import LogoIcon from '../Icons/LogoIcon';

const Link = styled(LinkBase)`
  height: 100%;
  cursor: pointer;
  padding: 0;
  position: relative;

  svg {
    max-width: 40rem;
  }
`;

const Wrapper = styled.header`
  background-color: ${colorDarkBlue};
  height: 8.5rem;
  padding: 0 4rem;
  grid-area: header;
`;

const Header = () => (
  <Wrapper>
    <Link to="/">
      <LogoIcon />
    </Link>
  </Wrapper>
);

export default Header;
