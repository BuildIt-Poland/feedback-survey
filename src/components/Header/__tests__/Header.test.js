import React from 'react';
import { create } from 'react-test-renderer';

import Header from '../Header';

jest.mock('react-router-dom', () => ({
  Link: 'Link'
}));

describe('COMPONENT - Header', () => {
  it('renders correctly', () => {
    const component = create(<Header />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
