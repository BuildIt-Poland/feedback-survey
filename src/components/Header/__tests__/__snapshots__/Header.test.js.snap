// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - Header renders correctly 1`] = `
<div
  className="StyledGrommet-sc-19lkkz7-0 eUfLDp"
>
  <div
    className="StyledBox-sc-13pk1d4-0 bNlcVt"
  >
    <img
      alt="randstad logo"
      src="/image/logo.svg"
    />
  </div>
</div>
`;
